transformations/transformations/single_column_base.py

```diff
@@ -275,7 +293,7 @@ def process_kernel(self, routine):
             return
 
         # check for horizontal loop bounds in subroutine symbol table
-        self.check_horizontal_var(routine, self.horizontal)
+        bounds = self.check_horizontal_var(routine, self.horizontal)
```

Comment:
Yes I agree with the name change, but I'll keep the routine as an argument because I would like to also print the routine name to have a more informative error message.